{
  "modified": "2024-06-25T13:34:05Z",
  "published": "2024-06-25T13:34:05Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "SNYK-PYTHON-COLORRAMMA-6513688"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "colorramma",
        "purl": "pkg:pypi/colorramma"
      },
      "versions": [
        "1.0.0"
      ]
    }
  ],
  "references": [
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/pypi-is-under-attack-project-creation-and-user-registration-suspended-heres-the-details-c3b6291d4579"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-COLORRAMMA-6513688"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "8ae7bacfde7e9c708c19ca6eddbe42eb7a53483f7e9f3e511b8f3ee4dbcabca6",
        "import_time": "2024-06-28T02:48:39.84249338Z",
        "id": "RLMA-2024-03755",
        "modified_time": "2024-06-25T13:34:05Z",
        "versions": [
          "1.0.0"
        ]
      }
    ]
  }
}
